### DISCUSSION TITLE
Allow users to search community scripts.

### DISCUSSION THREAD

**Discussion by stevebarakat** [johnlindquist/kit#591] [Ideas]
It would be nice to be able to search community scripts. 🔦

**Comment by johnlindquist** [comment]
If you filter by "share" in GitHub discussions (that's where all the community scripts are) then you can search there. I can bring something similar to the app I think 

I'm looking into putting an algolia search on the website. I'd really like a "search by api" someday.

**Reply by stevebarakat** [reply]
Good to know! Thanks for the tip. 
...speak of the devil, I just discovered Angolia. I'm using it for a headless WordPress site I have to make. It's pretty awesome.
